lib/charms/istio_beacon_k8s/v0/service_mesh.py

```diff
@@ -1065,10 +1071,19 @@ def __init__(
             logger=self.log,
         )
 
+    @staticmethod
+    def _get_all_supported_policy_resource_types() -> LightkubeResourceTypesSet:  # type: ignore
+        """Return all the resource types supported by the PRM class."""
+        # Flatten all resource types from all mesh types into one set
+        resource_types: LightkubeResourceTypesSet = set()  # type: ignore
+        for mesh_resource_types in POLICY_RESOURCE_TYPES.values():
+            resource_types.update(mesh_resource_types)
+        return resource_types
+
```

Comment:
This already exists :D 

```suggestion
    @staticmethod
    def _get_all_supported_policy_resource_types() -> LightkubeResourceTypesSet:  # type: ignore
        """Return all the resource types supported by the PRM class."""
        return set(RESOURCE_TYPES.values())

```